mothership-ec/cog-mothership-returns
# Return Slip Fix

#### What does this do?

Updates return slip to use `return.item` where necessary.
#### How should this be manually tested?

Waiting for https://github.com/messagedigital/cog-mothership-returns/pull/165 in. After that process a few returns, make sure all the numbers are correct and exchange items show. Do this for a client that does not have an override for return slips (in other words, don't test it on UW, because they have their own PR for this: https://github.com/messagedigital/uniform_wares/pull/599).
#### Related PRs / Issues / Resources?

Relies on https://github.com/messagedigital/cog-mothership-returns/pull/165 to fix document creation
UW's Return Slip Fix: https://github.com/messagedigital/uniform_wares/pull/599
#### Anything else to add? (Screenshots, background context, etc)
